app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -27,7 +27,18 @@ def simulate?
         Figaro.env.acuant_simulator == 'true'
       end
 
-      delegate :idv_session, to: :@context
+      def attempter
```

Comment:
This diff is pretty heavy. I think including changes like this, which aren't totally necessary for the change on the table in a separate PR would help slim things down and make this easier to review in the future.

Reply:
You said you would not approve it without the throttling.